crates/sui-core/src/authority/shared_object_congestion_tracker.rs

```diff
@@ -25,34 +26,33 @@ use sui_types::transaction::{Argument, SharedInputObject, TransactionDataAPI};
 pub struct SharedObjectCongestionTracker {
     object_execution_cost: HashMap<ObjectID, u64>,
     mode: PerObjectCongestionControlMode,
+    max_accumulated_txn_cost_per_object_in_commit: u64,
     gas_budget_based_txn_cost_cap_factor: Option<u64>,
+    max_txn_cost_overage_per_object_in_commit: u64,
 }
 
 impl SharedObjectCongestionTracker {
     pub fn new(
+        initial_object_debts: impl IntoIterator<Item = (ObjectID, u64)>,
         mode: PerObjectCongestionControlMode,
+        max_accumulated_txn_cost_per_object_in_commit: Option<u64>,
         gas_budget_based_txn_cost_cap_factor: Option<u64>,
+        max_txn_cost_overage_per_object_in_commit: u64,
     ) -> Self {
+        let max_accumulated_txn_cost_per_object_in_commit =
+            if mode == PerObjectCongestionControlMode::None {
+                0
```

Comment:
Nit: although this doesn't matter in the current implementation, probably more intuitive to set it to a large value.

Reply:
My thinking is that 0 makes it more likely for us to detect any kind of bug where we accidentally start enforcing congestion control when the mode is `None`